src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,43 @@
 import React from 'react';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+import { Post } from '../../types/Post';
+
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+
+import './PostInfo.scss';
+
+interface PostInfoProps {
```

Comment:
make it simple, just use `Props`, rename this in all cases
```suggestion
interface Props {
```